test/e2e/serial/tests/workload_placement.go

```diff
@@ -809,3 +813,30 @@ func matchLogLevelToKlog(cnt *corev1.Container, level operatorv1.LogLevel) (bool
 	val, found := rteFlags.GetFlag("--v")
 	return found, val.Data == kLvl.String()
 }
+
+func logSchedulerPluginLogs(fxt e2efixture.Fixture) {
```

Comment:
this reminds me that we need to review (someday in the future) the way we handle and pass around Fixture objects. Passing by value (not the pointer) seems most correct indeed.